README.md

```diff
@@ -122,19 +108,21 @@ between client-go versions.
 
 | Branch         | Canonical source code location       | Maintenance status            |
 |----------------|--------------------------------------|-------------------------------|
-| client-go 1.4  | Kubernetes main repo, 1.4 branch     | = -                           |
-| client-go 1.5  | Kubernetes main repo, 1.5 branch     | = -                           |
-| client-go 2.0  | Kubernetes main repo, 1.5 branch     | = -                           |
-| client-go 3.0  | Kubernetes main repo, 1.6 branch     | = -                           |
-| client-go 4.0  | Kubernetes main repo, 1.7 branch     | = -                           |
-| client-go 5.0  | Kubernetes main repo, 1.8 branch     | = -                           |
-| client-go 6.0  | Kubernetes main repo, 1.9 branch     | = -                           |
-| client-go 7.0  | Kubernetes main repo, 1.10 branch    | = -                           |
-| client-go 8.0  | Kubernetes main repo, 1.11 branch    | =-                            |
-| client-go 9.0  | Kubernetes main repo, 1.12 branch    | =-                            |
-| client-go 10.0 | Kubernetes main repo, 1.13 branch    | ✓                             |
-| client-go 11.0 | Kubernetes main repo, 1.14 branch    | ✓                             |
-| client-go 12.0 | Kubernetes main repo, 1.15 branch    | ✓                             |
+| `release-1.4`  | Kubernetes main repo, 1.4 branch     | = -                           |
+| `release-1.5`  | Kubernetes main repo, 1.5 branch     | = -                           |
+| `release-2.0`  | Kubernetes main repo, 1.5 branch     | = -                           |
+| `release-3.0`  | Kubernetes main repo, 1.6 branch     | = -                           |
+| `release-4.0`  | Kubernetes main repo, 1.7 branch     | = -                           |
+| `release-5.0`  | Kubernetes main repo, 1.8 branch     | = -                           |
+| `release-6.0`  | Kubernetes main repo, 1.9 branch     | = -                           |
+| `release-7.0`  | Kubernetes main repo, 1.10 branch    | = -                           |
+| `release-8.0`  | Kubernetes main repo, 1.11 branch    | =-                            |
+| `release-9.0`  | Kubernetes main repo, 1.12 branch    | =-                            |
+| `release-10.0` | Kubernetes main repo, 1.13 branch    | =-                            |
+| `release-11.0` | Kubernetes main repo, 1.14 branch    | ✓                             |
```

Comment:
this is about to not be maintained any more... do you want to drop this now or wait until the last 1.14.x release publishes later today?

Reply:
> do you want to drop this now or wait until the last 1.14.x release publishes later today?

I'd prefer to wait. I think it would make sense to turn down the godeps publishing-bot when CI for 1.14 is turned down as well, and if it takes O(month) from now (https://github.com/kubernetes/k8s.io/issues/500#issuecomment-564552627), it might be better to leave this here...

Reply:
As per the definition we are using here: ✓  = Changes in main Kubernetes repo are actively published to client-go by a bot